### PR TITLE
[ADD] account_invoice_refund_reinvoice

### PR DESCRIPTION
**Pull request by etobella**
This addon allows to restore a refund keeping all it's data.
It is useful when the invoice is linked to a Sale Order or Purchase Order, this way no information is lost.

Usage
=====

* Access a refund
* Press on `Restore Credit Note`
* A new invoice is created using the same information

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
Code review :+1:

File: account_invoice_refund_reinvoice/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "name": "Account Invoice Reinvoice Refund",
+    "summary": """
+        Allow to Reinvoice a Refund""",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Creu Blanca,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/account-invoicing",
+    "depends": ["account"],
+    "data": ["views/account_move.xml"],
+    "demo": [],
```
Comment (AaronHForgeFlow) [inline account_invoice_refund_reinvoice/__manifest__.py]: This can me removed I think

**Review by AaronHForgeFlow — COMMENTED** [review]
LGTM! Could you please add at least one unit test?

File: account_invoice_refund_reinvoice/__manifest__.py (hunk 1 above)
Comment (etobella) [reply account_invoice_refund_reinvoice/__manifest__.py]: Sure!

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM

**Review by olgamarcocb — APPROVED** [review]
:+1:

**Pull request merged**